regression/cbmc-java/stack_var4/test.desc

```diff
@@ -3,8 +3,6 @@ stack_test.class
 
 ^EXIT=0$
 ^SIGNAL=0$
-^.*assertion at file stack_test.java line 5 function.*: SUCCESS
```

Comment:
Why is this line (and others in cbmc-java) being removed? Should the test (ideally) be modified to stop it being trivially simplified?